src/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.jsx

```diff
@@ -316,12 +311,12 @@ const DotPlotPage = (props) => {
         <center>
           <Empty description={(
             <>
-              <Paragraph>
```

Comment:
have you checked that `Paragraph` styling is the same to `p` in all browsers? I'm concerned you checked that it looked good in your browser but `<Paragraph>` sets the styling for older or different browsers to make it look consistent everywhere.

Reply:
The antd `Paragraph` (https://github.com/ant-design/ant-design/blob/master/components/typography/Paragraph.tsx) replaces paragraphs with `div`s and adds a 1-em margin styling, which is also added for normal (non-antd) paragraph elements. I've checked that replacing this tag with paragraphs doesn't change anything.